integration-tests/src/test/java/oracle/kubernetes/operator/ItUsabilityOperatorHelmChart.java

```diff
@@ -36,11 +37,11 @@
   private static StringBuffer namespaceList;
 
   /**
-   * This method gets called only once before any of the test methods are executed. It does the
-   * initialization of the integration test properties defined in OperatorIT.properties and setting
-   * the resultRoot, pvRoot and projectRoot attributes.
+   * This method gets called only once before any of the test methods are executed
+   * It does the initialization of the integration test properties defined in 
+   * OperatorIT.properties and define resultRoot,pvRoot and projectRoot attributes.
    *
-   * @throws Exception exception
+   * @throws Exception if the test initilization fails
```

Comment:
spelling.... stopping here.

Reply:
The correct spelling is, "initialization".  @anpanigr, I would suggest turning on spell checking in your IDE.

Reply:
@throws Exception if the test initialization fails